sample-queries-tpch/tpch_query2.sql

```diff
@@ -1,4 +1,4 @@
-drop view q2_min_ps_supplycost;
+${EXPLAIN} drop view q2_min_ps_supplycost;

```

Comment:
nice catch!. I used an automated script to add explain to the begining of query files. Corrected this now.